### ISSUE TITLE
Crash on malformed code

### ISSUE THREAD

**Issue by GLN** [ccrma/chuck#39]
This charming snippet of malformed code will crash ChucK 1.3.5.0 (and miniAudicle with it): 

``` chuck
SinOsc s => dac;
[s] => SinOsc a[];
```

**Comment by spencersalazar** [comment]
Thanks! Shouldn't be hard to track down.

**Comment by gewang** [comment]
(finally) resolved in 6251ea6; using `=>` to connect from/to empty UGen array declaration no longer
    results in a crash; instead there is now a compiler error:
```
test.ck:8:17: error: cannot connect '=>' to empty array '[ ]' declaration...
[8] [s] => SinOsc a[];
                  ^
test.ck: ...(hint: declare 'a' as an non-empty array)
test.ck: ...(or if assignment was the intent, use '@=>' instead)
```
